### DISCUSSION TITLE
Getting information about a repo's test coverage from Drone builds

### DISCUSSION THREAD

**Discussion by yaelleC** [grafana/grafana#45959] [Observability]
_This is a Feature Request for the release team_

## Background
- I have a dashboard that shows test coverage for our team's repos: https://ops.grafana.net/d/3kxjo4fnz/cloud-datasource-unit-test-coverage?orgId=1&from=now-7d&to=now

- For now, most steps about data collection are manual (using a google sheet as datasource). 

- Drone build logs have information about test coverage (at least for external plugins - see bottom of previous dashboard). I would love to be able to use that as input for the dashboard

## Request/Questions

- [ ] Could we emit a metric from the drone build instead of scrapping the logs? Getting a new datapoint every time a build passes on the main branch would be ideal.
- [ ] Could we have that information (from metric or logs) for `grafana/grafana` too? So far I have not been able to find it

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
